workflow/hosts/s4.yaml

```diff
@@ -23,5 +23,5 @@ ATARDIR: '${NOSCRUB}/archive_rotdir/${PSLOT}'
 MAKE_NSSTBUFR: 'YES'
 MAKE_ACFTBUFR: 'YES'
 SUPPORTED_RESOLUTIONS: ['C384', 'C192', 'C96', 'C48']
-AERO_INPUTS_DIR: /data/prod/glopara/gocart_emissions
+AERO_INPUTS_DIR: /data/prod/glopara/GEFS_ExtData/20250310
```

Comment:
Similar to Jet, this doesn't exist and will go away as we drop support for S4 soon. Just an FYI.